modules/administration-guide/partials/proc_migration_olm_stable_channel_to_stable_all_namespace.adoc

```diff
@@ -0,0 +1,271 @@
+
+[id="migration-olm-stable-channel-to-stable-all-namespace_{context}"]
+= Migration {prod} from "che-server" workspace engine to "dev-workspace" using OLM 
+
+For {prod} instances installed using Operator lifecycle manager(OLM) with "stable" channel
+there is opportunity to make migration with new channel "tech-preview-stable-all-namespaces".
+"stable" OLM channel provides {prod} with old "che-server" engine.
+This engine is not under active development any more.
+"tech-preview-stable-all-namespaces" channel provides {prod-short} with modern "dev-workspace" engine.
+
+> Warning: Migration process is not back compatible. All previously created workspaces will be
+not working any more after migration. Before this procedure all users should be notified to save their source code
+changes to prevent losing data.
+
+.Prerequisites
+
+* The `{orch-cli}` tool is available.
+* An instance of {prod-short} running in Openshift 4 cluster.
+
+.Procedure
+
+Scale {prod-deployment} deployment to zero to stop users interaction with this service:
+
+[subs="+quotes,+attributes"]
+----
+{orch-cli} scale deployment {prod-deployment} --replicas=0 -n {prod-namespace}
+----
+
+If you have enabled Openshift OAuth you have to migrate existed users. This migration is required to reuse existed Openshift users with "dev-workspace" native authentication mode.
+
+> Warning: If your {prod} instance didn't use OAuth, then you can't migrate users.
+There is no mechanism to migrate native Keycloak users to Openshift users.
+
+## Migrate Openshift OAuth users
+
+Create {prod} backup using the xref:managing-backups-using-chectl.adoc[] or che-operator itself xref:managing-backups-using-custom-resources.adoc[].
+
+Create migration script MigrateUsers.sh:
+
+[subs="+quotes,+attributes"]
+----
+$ touch MigrateUsers.sh && chmod +x MigrateUsers.sh
+----
+
+MigrateUsers.sh script content
```

Comment:
wondering if we need to store the script in the repo somewhere

Reply:
It is better to store script in docs since it automatically replace some values on downstream specific